src/index.html

```diff
@@ -11,7 +11,144 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="#" class="form"
```

Comment:
Checklist point 3 and also please use the link from The Task section in readme.md :)